src/GitVersionCore/BuildServers/TravisCI.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+namespace GitVersion
+{
+	public class TravisCI : BuildServerBase
+	{
+		public const string EnvironmentVariableName = "TRAVIS";
+
+		public override bool CanApplyToCurrentContext ()
+		{
+			return "true".Equals(Environment.GetEnvironmentVariable(EnvironmentVariableName)) && "true".Equals(Environment.GetEnvironmentVariable("CI"));
```

Comment:
We have Fody.Caseless running which makes everything automatically case insensitive.